.gitignore

```diff
@@ -38,3 +41,7 @@ subprojects/proxy-libintl*
 subprojects/wxWidgets
 subprojects/zlib-*
 subprojects/dirent-*
+subprojects/expat-*
+subprojects/googletest-*
```

Comment:
What subproject is bringing these in? I don't think we should actually need to exclude them, but maybe I'm forgetting a configuration.

Reply:
`expat` never used, I'll remove it.
`googletest` used in `test/meson.build`, but this is a part of another pr. I'll remove it first, and then add it back.